deploy/crds/eunomia.kohls.io_gitopsconfigs_crd.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
If this is the new CRD yaml file then the old CRD yaml file, eunomia_v1alpha1_gitopsconfig_crd.yaml, should be deleted.